test/boulder-config.json

```diff
@@ -323,7 +323,6 @@
   },
 
   "common": {
-    "baseURL": "http://127.0.0.1:4000",
```

Comment:
I think this should be in a separate PR if we're ready to remove it from `boulder-config.json`